ci-constraints-requirements.txt

```diff
@@ -253,13 +253,9 @@ tomli==2.2.1 ; python_full_version <= '3.11'
     #   nox
     #   pytest
     #   sphinx
-typing-extensions==4.13.2 ; python_full_version < '3.9'
-    # via
-    #   exceptiongroup
-    #   mypy
-    #   virtualenv
-typing-extensions==4.14.1 ; python_full_version >= '3.9'
+typing-extensions==4.13.2
```

Comment:
This file should be built in a way that uses the highest version for each python version:

`uv pip compile -U --universal --python-version 3.8 --all-extras --unsafe-package=cffi --unsafe-package=pycparser --unsafe-package=setuptools --unsafe-package=cryptography-vectors --unsafe-package=bcrypt pyproject.toml -o ci-constraints-requirements.txt -q` is the right way to build it.